### ISSUE TITLE
srli.S - no reference_output file

### ISSUE THREAD

**Issue by ad-sc** [riscv/riscv-compliance#17]
srli.S test included in rv32ui set doesn't have it's 'reference_output' file and also there's no error message during signature comparison.

**Comment by eroom1966** [comment]
good catch, I have added a reverse check to see if a signature is generated for a reference that does not exist, I am also renaming the output signature to be in sympathy with the naming, ie
<stub>.reference_output
<stub>.signature_output

**Comment by eroom1966** [comment]
Hi Neel
if this is fixed can you close ?
Thx
Lee

**Comment by neelgala** [comment]
Yes the pull request was merged successfully. 
I can't close since I am not an admin on the repo.
Only owners can close the issue.
